### DISCUSSION TITLE
Postal Server not Sending to Gmail

### DISCUSSION THREAD

**Discussion by rolix202** [postalserver/postal#2113] [Help with using Postal]
This is the error message i got when i trying sending to Gmail, others are working fine except gmail

550-5.7.1 [94.250.203.54 12] Our system has detected that this message is 550-5.7.1 likely unsolicited mail. To reduce the amount of spam sent to Gmail, 550-5.7.1 this message has been blocked. Please visit 550-5.7.1 https://support.google.com/mail/?p=UnsolicitedMessageError 550 5.7.1 for more information. b11-20020adfd1cb000000b0021ed3ae5a34si292424wrd.53 - gsmtp

**Comment by reetikpant** [comment]
thats coz you have spammed so many mails, what i do? reinstall smtp again

**Reply by rolix202** [reply]
okay, will try reinstall it again

**Comment by willpower232** [comment]
gmail is as gmail does I'm afraid, you could try mail-tester.com to see if there are any aspects of your messages which could be improved or corrected. You can also check blocklists with mxtoolbox.com in case you're on there.
